ReactViewResources/Loader.ts

```diff
@@ -80,9 +80,14 @@ function loadScript(scriptSrc: string): Promise<void> {
 }
 
 async function loadFramework(): Promise<void> {
-    /* React */ await loadScript(ExternalLibsPath + "react/umd/react.production.min.js");
-    /* ReactDOM */ await loadScript(ExternalLibsPath + "react-dom/umd/react-dom.production.min.js");
-    /* Prop-Types */ await loadScript(ExternalLibsPath + "prop-types/prop-types.min.js");
+    let frameworkPromises: Promise<void>[] = [];
+    frameworkPromises.push(
+        /* React */ loadScript(ExternalLibsPath + "react/umd/react.production.min.js"),
+        /* ReactDOM */ loadScript(ExternalLibsPath + "react-dom/umd/react-dom.production.min.js"),
+        /* Prop-Types */ loadScript(ExternalLibsPath + "prop-types/prop-types.min.js")
+    );
+    await Promise.all(frameworkPromises);
+
     /* React View Resources */ await loadScript(LibsPath + "ReactViewResources.js");
```

Comment:
/* React View Resources */ move to the end of the line (messes up with indentation), do the same for the ones above